### ISSUE TITLE
Access List of friends

### ISSUE THREAD

**Issue by obydog002** [FAForever/downlords-faf-client#3168]
**Is your feature request related to a problem? Please describe.**
Currently, it is only possible to see friends when they are online. It would be nice to see all friends online or offline.  

This would make it easier to remove friends also.

**Describe the solution you'd like**

This could be via a separate screen or modal.

**Comment by obydog002** [comment]
Heres two mockups:

First allow an option to display offline friends in the chat 

![friendPopu](https://github.com/user-attachments/assets/ee0b353e-ce51-4b2c-b2e5-efa2b243ea46)

If showing offline friends they could appear in a different color like so:

![mock1](https://github.com/user-attachments/assets/6079fc85-cdc4-4dab-b36d-cef70e7b8498)

**Comment by obydog002** [comment]
Second is to have a separate tab in the profile view

![friendList](https://github.com/user-attachments/assets/6e98dd68-b268-4602-8a02-ee5f00b3a571)

Clicking them would have the same options as in the replay views or a few other places.

@Sheikah45  @Brutus5000  thoughts? Or other ideas? It may also be worth considering the same functionality for foes

**Comment by Sheikah45** [comment]
I think that having them in a separate area from the chat list would be preferable. And I agree a similar thing should be done for foes.

**Comment by Brutus5000** [comment]
Good idea. A common requested feature was also to add friends that are not online and to delete foes that are not online. This could be done here as well.

**Comment by sulikdan** [comment]
Hmmm, wouldn't it be better to have it already in chat area?
E.g:
- Friends
-- Online (Nested - Rolled/Spread out )
-- Offline (Nested - Collaped)
- Foe
-- Online ( nested - Rolled/Spread out)
-- Offline ( nested - collaped)

Would do this only for specific categories and not for the whole chat(categories). Anyway, don't know how hard it is to implement :X. Easier to be said that done.

**Comment by Sheikah45** [comment]
It should not go in the chat area because then it would have to go in every channel a player has joined and that doesn't make much sense. Their isn't technically a main channel for FAF so it can't go in one place. Additionally if we place it in the chat area then players have to be connected to chat to access it where as placing it outside of chat enables access all the time. And could set the place for just seeing all online players not just those connected to the chat.

**Comment by obydog002** [comment]
I will get to this within the month

**Comment by sulikdan** [comment]
@obydog002 good luck, was looking into it, but with my miserable JavaFx knowledge, I had no idea, where is the setting that needs to be changed..
